metricsmentor/templates/base.html

```diff
@@ -48,6 +48,19 @@
         <link rel="icon" href="{{STATIC_URL}}img/favicon.ico">
         <link rel="icon" type="image/png" href="{{STATIC_URL}}img/favicon.png">
 
+        <meta id="csrf-token" name="csrf-token" content="{{csrf_token}}">
+        <script>
+            window.MetricsMentor = {
+                staticUrl: '{{ STATIC_URL }}',
+                baseUrl: '//{{ request.get_host }}',
+                {% if not request.user.is_anonymous %}
+                currentUser: {
+                    id: '{{ request.user.id }}',
```

Comment:
Hmmm, not a big deal, but I think this might as well be treated as a number when converting to javascript:
```
id: {{ request.user.id }},
```

Thought, maybe the string is a bit more safe if the id happens to be `None` for some reason in python. But, you already have the is_anonymous check.